deploy/README.md

```diff
@@ -304,6 +302,41 @@ To run the pipeline, open the Synapse Studio for the Synapse workspace that you
 
 - Wait for the pipeline to complete.
 
+# Running the pipeline (Custom Vision Model V2)
```

Comment:
Do we want to repeat these contents for both pipelines ? Versioning / update at both places challenges later. Can we consolidate this section to run both the pipelines and call out deltas

Reply:
Discussed this section with Mandar.